.github/workflows/e2e-staging.yml

```
@@ -0,0 +1,53 @@
+name: E2E Staging Tests
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)